src/test/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryTaskLauncherIntegrationTests.java

```diff
@@ -110,7 +110,7 @@ public void init() {
 
 		CloudFoundryOperations cloudFoundryOperations = DefaultCloudFoundryOperations.builder()
```

Comment:
This should not be needed. The taskDeployer should be injected and come from the "real, production" AutoConf triggered by `@SpringApplicationConfiguration(classes = CloudFoundryTaskLauncherIntegrationTests.Config.class)`